src/components/CaptionExtension.tsx

```diff
@@ -84,7 +84,43 @@ const mutation = `
   }
 `;
 
-let captionError = undefined;
+function PreviewButton() {
+  return (
+    <Link
+      href="https://amplience.com/developers/docs/knowledge-center/amplience-labs"
+      underline="none"
+      variant="link"
+    >
+      Amplience Labs Preview
+    </Link>
+  );
+}
+
+function CaptionError({ error }: { error: any }) {
+  if (error?.errors[0]?.extensions?.code === "INSUFFICIENT_CREDITS") {
+    return (
+      <>
+        You're out of Amplience Credits. You can still enter alt text yourself.{" "}
+        <a
```

Comment:
This should be a `Link` component with `variant="link" underline="none"` for styling consistency